opensoundscape/seed.py

```diff
@@ -0,0 +1,15 @@
+""" seed.py: Set random state across different libraries for reproducibility
+"""
+import numpy as np
+import torch
+import random
+
+
+def seed(seed, verbose=True):
+    print(f"Random state set with seed {seed}")
```

Comment:
did you mean to wrap this in `if verbose:` ? `verbose` is currently unused

Reply:
Yes, I did. Just changed that.